src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -71,17 +71,23 @@ public function search()
     {
         $this->crud->hasAccessOrFail('list');
 
+        // should the `Showing 1 of 10 ... ` be displayed ? When it's not displayed we wouldn't do the counting calculations.
+        $showEntryCount = $this->crud->getOperationSetting('showEntryCount');
+
+        // how many total entries there is applying the controller/model constrains, before any filtering or search
+        $unfilteredQueryCount = request('unfilteredQueryCount') ?? $this->crud->getOperationSetting('unfilteredQueryCount');
+
+        $totalRows = ! $showEntryCount ? 0 : $unfilteredQueryCount ?? $this->crud->getQueryCount();
+
+        $this->crud->setOperationSetting('unfilteredQueryCount', $totalRows);
```

Comment:
Same feedback as in the `CrudFilter` file. Aaaand... it looks like it's the exact same calculation, isn't it? In that case, it probably makes sense to move that logic to its own function, something like `$this->crud->setUnfilteredQueryCount()` or `$this->crud->setUnfilteredQueryCount()`. Right?